util/dvsim/Testplan.py

```diff
@@ -15,13 +15,41 @@
 from tabulate import tabulate
 
 
+class JobTime:
+    # Possible units.
+    units = ["h", "m", "s", "ms", "ns", "us", "ps", "fs"]
+
+    def __init__(self):
+        self.time = 0.0
+        self.unit = "s"
+
+    def __str__(self):
+        if self.time == 0:
+            return ""
+
+        try:
+            index = self.units.index(self.unit)
+        except ValueError as e:
+            log.debug("Unsupport JobTime unit: " + self.unit + f"\n{e}")
+            return ""
+
+        normalized_time = self.time
+        while index > 0 and normalized_time >= 1000:
+            normalized_time = normalized_time / 1000.0
+            index = index - 1
```

Comment:
For hours / minutes / seconds, the divider is 60, not 1000.

Reply:
wow! Good catch!